quelpa.el

```diff
@@ -472,6 +479,8 @@ the global var `quelpa-upgrade-p' is set to nil."
   (when (quelpa-setup-p)
     ;; shadow `quelpa-upgrade-p' taking the default from the global var
     (let* ((quelpa-upgrade-p (if current-prefix-arg t quelpa-upgrade-p))
+           ;; shadow `quelpa-stable-p'
+           (quelpa-stable-p quelpa-stable-p)
```

Comment:
How exactly does "shadowing" work here? Let-binding a variable to its value seems kind of silly.